backend/src/main/scala/cromwell/backend/standard/callcaching/StandardCacheHitCopyingActor.scala

```diff
@@ -209,10 +223,14 @@ abstract class StandardCacheHitCopyingActor(val standardParams: StandardCacheHit
 
   when(FailedState) {
     case Event(f: IoReadForbiddenFailure[_], Some(data)) =>
-      handleForbidden(
+      handleBlacklistingForForbidden(
         path = f.forbiddenPath,
         andThen = stayOrStopInFailedState(f, data)
       )
+    case Event(fail: IoFailAck[_], Some(data)) =>
+      // Not a forbidden failure so do not blacklist the bucket but do blacklist the hit.
+      handleBlacklistingForGenericFailure()
```

Comment:
This comment probably belongs inside the `handleBlacklistingForGenericFailure ` method rather than at the point it gets called?